algebird-util/src/main/scala/com/twitter/algebird/util/UtilAlgebras.scala

```diff
@@ -52,11 +52,9 @@ object UtilAlgebras {
   implicit def futureMonoid[T: Monoid]: Monoid[Future[T]] = new ApplicativeMonoid[T, Future]
   implicit def futureGroup[T: Group]: Group[Future[T]] = new ApplicativeGroup[T, Future]
   implicit def futureRing[T: Ring]: Ring[Future[T]] = new ApplicativeRing[T, Future]
```

Comment:
Right. I assume that these are here for backwards compatibility?

(We do have `algebra.ring.Semiring` if you wanted to be more precise, although you might still not want to define `exception * 0` as `0`.)